<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KGMain</class>
 <widget class="QMainWindow" name="KGMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KGMain</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AnimatedDocks|QMainWindow::VerticalTabs</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="TaskListForm" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="dockProject">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>261</width>
     <height>296</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetVerticalTitleBar</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::NoDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Keepgoing</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="3" column="0">
      <widget class="QGroupBox" name="pProject">
       <property name="title">
        <string>Projekte</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLineEdit" name="lineEdit">
          <property name="placeholderText">
           <string>Neues Projekt</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QListView" name="projectView"/>
        </item>
        <item row="0" column="1">
         <widget class="QToolButton" name="btPrjOk">
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Focus</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="inboxButton">
          <property name="text">
           <string>Inbox</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="starredButton">
          <property name="acceptDrops">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Starred</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TaskListForm</class>
   <extends>QWidget</extends>
   <header>tasklistform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
